src/components/preset-control/preset-control-list/preset-control-list.tsx

```diff
@@ -0,0 +1,92 @@
+import styles from './preset-control-list.module.css'
+
+import { Fragment, ReactElement, useState } from 'react'
+
+import { t } from '@i18n'
+import { Icon } from '@ui/icon'
+import { MenuList, MenuListItem } from '@ui/menu-list'
+import { Popover } from '@ui/popover'
+import { ISolutionEntryDescription } from '@service-providers/common'
+
+interface IPresetControlList {
+  className?: string
+  presets: ISolutionEntryDescription[]
+  selected?: string
+  onSelect: (presetName: string) => void
+  onModify: (presetName: string) => void
+  onDelete: (presetName: string) => void
+}
+
+export const PresetControlList = ({
+  className,
+  presets,
+  selected,
+  onSelect,
+  onModify,
+  onDelete,
+}: IPresetControlList): ReactElement => {
+  const [contextMenuItem, setContextMenuItem] = useState<{
+    name: string
+    element: HTMLElement
+  } | null>(null)
+
+  const closeContextMenu = () => {
+    setContextMenuItem(null)
+  }
+
+  return (
+    <Fragment>
```

Comment:
In some places you use `<Fragment>` in other `<>`, maybe we should use standard variant in hole project

Reply:
I think we should create a vote for it and use as code style guideline

Reply:
Fixed to `<>`